api/src/logger.ts

```diff
@@ -24,7 +24,7 @@ const loggerEnvConfig = getConfigFromEnv('LOGGER_', 'LOGGER_HTTP');
 if (loggerEnvConfig.levels) {
 	const customLogLevels: { [key: string]: string } = {};
 
-	for (const el of loggerEnvConfig.levels.split(',')) {
+	for (const el of loggerEnvConfig.levels) {
```

Comment:
Think we should be wrapping it in `toArray()` as it will be of `string` type if there's only one level.